nephio-controllers/app/controller/deployment-controller.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations: {}
+  creationTimestamp: null
+  name: nephio-controller
+  namespace: nephio-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      fn.kptgen.dev/controller: nephio-controller
+  strategy: {}
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app.kubernetes.io/component: tbd
+        app.kubernetes.io/instance: tbd
+        app.kubernetes.io/managed-by: kpt
+        app.kubernetes.io/name: nephio
+        app.kubernetes.io/part-of: nephio
+        app.kubernetes.io/version: tbd
+        fn.kptgen.dev/controller: nephio-controller
+      name: nephio-controller
+      namespace: nephio-system
+    spec:
+      containers:
+      - args:
+        - --secure-listen-address=0.0.0.0:8443
+        - --upstream=http://127.0.0.1:8080/
+        - --logtostderr=true
+        - --v=10
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+        name: kube-rbac-proxy
+        ports:
+        - containerPort: 8443
+          name: https
+        resources: {}
+      - args:
+        - --health-probe-bind-address=:8081
+        - --metrics-bind-address=127.0.0.1:8080
+        - --leader-elect
+        command:
+        - /manager
+        env:
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
+        - name: POD_IP
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: status.podIP
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.name
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: spec.nodeName
+        - name: NODE_IP
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: status.hostIP
+        - name: GIT_URL
+          value: http://172.18.0.200:3000
```

Comment:
I see, this is hardcoded for now, you mentioned that. OK, that's fine for now, but we'll have to think about how we want that to work in the long run. Some nephio config resource in-cluster, probably.